### DISCUSSION TITLE
.env.development.local 꼭 개인으로 다 만들기!

### DISCUSSION THREAD

**Discussion by soeunl** [project34plus/project5_config#15] [Ideas]
만들지 않으면 무시무시한 일이 일어남
오타가 나도 무시무시한 일이 일어남

위치!
![라인](https://github.com/user-attachments/assets/e5489058-9599-4bec-b4e1-1c91cad19607)
![image](https://github.com/user-attachments/assets/08c3dff4-706d-43b4-8276-bfaacc40985d)

![image](https://github.com/user-attachments/assets/401b5f96-7765-4cb4-a75f-20a293c2a227)
#NEXT_PUBLIC_API_URL=http://192.168.0.39:5000/api/v1
NEXT_PUBLIC_API_URL=http://joyfarm.xyz:5000/api/v1
NEXT_PUBLIC_ADMIN_URL=http://localhost:6007

![image](https://github.com/user-attachments/assets/060583a4-2c6c-418c-9fba-da9ae21ab5bf)
NEXT_PUBLIC_API_URL=http://joyfarm.xyz:5000/api/v1
NEXT_PUBLIC_DOMAIN=nonnull.life

**Comment by hyeon12** [comment]
무서워요
